### DISCUSSION TITLE
Alphabetical ordering in "StreamLD", "StreamLL", and "StreamLT"

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/evm-monorepo#868] [Ideas]
Is there any reason for _not_ ordering the fields alphabetically in the `StreamLD`, `StreamLL`, and `StreamLT` structs, @smol-ninja, @andreivladbrg?

As I understand it, these structs are used only in-memory in the `getStream` getter function.

**Comment by smol-ninja** [comment]
The reason is to maintain consistencies in related structs.

From [this comment](https://github.com/sablier-labs/v2-core/pull/784#issuecomment-1879977129):

> By following the same order, as declaring other structs such as `CreateWithDurations`, throughout the codebase, it appears to make code easier to understand at a glance and reduce the likelihood of missing out on any struct variable since they are now in the same order. 

Similar order pattern in [Periphery's DataTypes](https://github.com/sablier-labs/v2-periphery/blob/staging/src/types/DataTypes.sol) too.

Does that answer?

**Reply by PaulRBerg** [reply]
Oh, I forgot about that discussion. Thanks for linking.

All clear now.
